src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,38 @@
 package core.basesyntax;
 
-public class SalaryInfo {
-    public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+import java.time.format.DateTimeParseException;
+
+public class SalaryInfo extends DataComparison {
```

Comment:
why do you extend datacomparison? it looks like just a utility class, make its method static and use directly